package/CHANGELOG

```diff
@@ -44,6 +44,19 @@ Enhancements
    `n_initial_contacts` attribute, with documentation. (Issue #2604, PR #4415)
  * Implement average structures with iterative algorithm from 
    DOI 10.1021/acs.jpcb.7b11988. (Issue #2039, PR #4524)
+ * Removed type and mass guessing from all topology parsers (PR #3753)
```

Comment:
Sorry to ask this, but since there are changelog conflicts to deal with anyways. Could you move this to these changes to the top of the list? (i.e. it should be "newest first")